### DISCUSSION TITLE
muCommander create dot files

### DISCUSSION THREAD

**Discussion by photocoder** [mucommander/mucommander#1288] [General]
Hello everyone,
Could you tell me what's going on?

Every time when I create a folder, it creates a copy with a "dot" file.
![dot-file-screenshot2](https://github.com/user-attachments/assets/b4708833-d5bc-46d3-bb95-cc2811d54b32)

If I copy files from my SD card to a hard drive, then each file has his "brother" with dot.
![dot-file-screenshot](https://github.com/user-attachments/assets/c2c8aa14-08e9-47b3-a441-1488d6acd3fc)

How can I avoid it? 
Thank you

**Comment by pskowronek** [comment]
https://apple.stackexchange.com/questions/14980/why-are-dot-underscore-files-created-and-how-can-i-avoid-them

https://superuser.com/questions/212896/is-there-any-way-to-prevent-a-mac-from-creating-dot-underscore-files

**Reply by photocoder** [reply]
Those are not solutions. It's the same as saying, "This manager created dot-files, but you just remove them; don't worry".

**Unfortunately, It's the bug of muCommander.**
These files are created ONLY by muCommander.
Any other different managers do not create them.

**Reply by ghost** [reply]
There's no bug in muCommander here. Most other file managers just don't show ._ files, even though they exist. That's a weakness of those other file managers.

**Reply by pskowronek** [reply]
> **Unfortunately, It's the bug of muCommander.** These files are created ONLY by muCommander. Any other different managers do not create them.

Are you sure? You have two links showing that they are being created by macOS itself for non-native filesystems like FAT or windows shared resources.

Go to a folder where you see those dot files using Finder, even if you have 'show hidden files' turned on, you won't see those dot files in Finder, but open Terminal, and issue the following command: `ls -la /path/to/the/folder` - the dot file will most probably be there.

Check you muCommander settings - if you don't want to see them, then change these settings :
<img width="351" alt="image" src="https://github.com/user-attachments/assets/1e845601-14e9-4e9f-8e4b-7536686dc183">

Check in 'Any other different managers' what similar settings you have there - most probably you have "show/hide hidden files" setting there.

**Reply by ahadas** [reply]
@photocoder as noted above, muCommander doesn't create those dot-files, they are created by macOS
version 1.5.2 doesn't show hidden files by default, so new installations won't show these files as other files managers you checked, I suppose, but you can also disable showing hidden files from the View menu (or as @pskowronek showed above).
however, I wonder if it would make sense to skip copying hidden files when they are not presented - can you please check what happens with other file managers you have when not showing hidden files and copying a folder with hidden files? (in other words, create a folder, make sure it has a .DS file, disable showing hidden files, copy the folder, navigate to the target folder, enable showing hidden files -> and see if you get the .DS file there)
